lib/seattleflu/db/cli/command/etl/presence_absence.py

```diff
@@ -0,0 +1,385 @@
+"""
+Process presence-absence tests into the relational warehouse
+
+The ETL process performs a `find_or_create` for targets, as we anticipate the same targets to be tested, but leave room for the addition of new targets.
+
+The ETL process performs a `find_or_create` for samples to allow the processing of re-tested samples. We do not expect to update information of a sample itself.
+
+The ETL process performs an `upsert` for presence_absence to allow the update of results should there be a re-test on an old sample.
+
+The presence absence ETL process will abort under these conditions:
+1. If a sample's barcode matches with a UUID that is of the incorrect identifier set
+2. If we receive an unexpected value for the "controlStatus" of a target
+3. If we receive an unexpected value for the "targetResult" of a specific test
```

Comment:
When the command is run with `--help`, the list here is all smooshed together. Best to format it for as-is presentation with line breaks in between, I think?